engine/resources/exec.go

```diff
@@ -138,9 +141,38 @@ func (obj *ExecRes) Validate() error {
 		}
 	}
 
+	// check that environment variables' format is valid
+	for key := range obj.Env {
+		if !isNameValid(key) {
+			return fmt.Errorf("environment variable name must only contain alphanumeric characters or underscore")
```

Comment:
errwrap.Wrapf please with shorter message now that below sig changes

Reply:
I don't quite understand what you mean by "now that below sig changes"

Reply:
Sorry, I get it now